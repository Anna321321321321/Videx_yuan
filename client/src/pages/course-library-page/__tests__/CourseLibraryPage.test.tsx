import { shallow } from 'enzyme';
import React from 'react';
import CourseLibraryPage from '..';

describe('CourseLibraryPage Unit Tests', () => {
  it('Shallow Render', () => {
    const wrapper = shallow(<CourseLibraryPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
