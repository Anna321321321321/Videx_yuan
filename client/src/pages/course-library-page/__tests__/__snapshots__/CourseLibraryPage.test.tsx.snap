// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourseLibraryPage Unit Tests Shallow Render 1`] = `
<Adapter
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <Component />
  <Connect(CourseLibraryContainer) />
  <Component />
</Adapter>
`;
